lib/SetAPI/assembly/AssemblySetInterfaceV1.py

```diff
@@ -55,15 +55,7 @@ def save_assembly_set(
         """
         self.set_interface._check_save_set_params(params)
         self._validate_save_set_params(params)
-
-        save_result = self.set_interface.save_set(
-            self.set_type(), ctx["provenance"], params
-        )
-        info = save_result[0]
-        return {
-            "set_ref": info_to_ref(info),
-            "set_info": info,
-        }
```

Comment:
Moved this transformation into the SetInterfaceV1 class so it isn't repeated in every file!